frontend/src/styles.scss

```diff
@@ -14,3 +14,24 @@
 .ol-control {
     width: fit-content !important;
 }
+
+// Controls the size of images
+.preview-image-container {
```

Comment:
Consider removing this from `frontend/src/app/pages/exercises/exercise/shared/trainer-map-editor/trainer-map-editor.component.scss`.